articles/azure-monitor/app/opencensus-python-dependency.md

```diff
@@ -13,6 +13,8 @@ A dependency is an external component that is called by your application. Depend
 
 First, instrument your Python application with latest [OpenCensus Python SDK](./opencensus-python.md).
 
+You can find sample dependency applications in the Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).
```

Comment:
I suggest putting this sentence before the previous line...`"First, instrument..."`

Reply:
All the "sample" wording was moved to the bottom of the pages.